docker/postgres-tools-9.6/Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM postgres:14-bullseye
-
-LABEL maintainer="vkamra@kasten.io"
+FROM postgres:15-bullseye
```

Comment:
As a follow-up, we should remove this Dockerfile and stop building this image through goreleaser. I don't think we use this anymore